### PR TITLE
Add "predict orthologs" capability

### PR DESCRIPTION
**Pull request by KristinaKa**
I get an error when trying to choose the target taxa, more comments in the script.

### CODE REVIEW

File: predict_orthologs.py [hunk 1]
```diff
@@ -109,66 +109,138 @@ def find_orthologs_per_hit(arguments):
...
+    from ete3 import NCBITaxa
+    ncbi = NCBITaxa()
+    ncbi.update_taxonomy_database()
+    for taxon in target_taxa:
+        taxid = ""
+        try:
+            taxid = int(taxon)
+            taxon = ncbi.get_taxid_translator([taxid])[taxid]
+        except ValueError:
+            taxid = ncbi.get_name_translator([taxon])[taxon]
```
Comment (jhcepas) [inline predict_orthologs.py]: This line returns a list, you need to take the first element: 
```taxid = ncbi.get_name_translator([taxon])[taxon][0]```

File: predict_orthologs.py [hunk 2]
```diff
@@ -109,66 +109,138 @@ def find_orthologs_per_hit(arguments):
...
+    target_taxa = args.target_taxa
+    if target_taxa == "all":
+        all_orthologs = annota.get_member_orthologs(best_hit_name)
+        return [(query_name, all_orthologs, "All", "All", best_hit_name)]
+    else:
+        results = []
+        for taxid, name in target_taxa:
+            species = target_taxa[(taxid,name)]
+            all_orthologs = annota.get_member_orthologs(best_hit_name, target_taxa=species)
+            results.append(query_name, all_orthologs, name, taxid, best_hit_name)
```
Comment (jhcepas) [inline predict_orthologs.py]: Missing brackets: `results.append([query_name, all_orthologs, name, taxid, best_hit_name])`

File: predict_orthologs.py [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
-from emapper import setup_hmm_search, dump_hmm_matches, iter_hit_lines, get_seq_hmm_matches, refine_matches
+from emapper import setup_hmm_search, dump_hmm_matches, get_seq_hmm_matches, refine_matches
 from ete3 import NCBITaxa
 
 # Example run: python predict_orthologs.py -i test/testCOG0515.fa -d bact
+
 # working options:  --output_format --orthology_type
 # not working: --target_taxa
 
 # TO DO:
```
Comment (jhcepas) [inline predict_orthologs.py]: There was a bug in annota.py. Fixed now in the master branch.

**Comment by jhcepas** [comment]
@KristinaKa did you add something in your last commit that needs to be integrated? 
If not, all functionality should now be included in the emapper.py and orthology.py. We can remove predict_orthologs.py I guess.

**Comment by KristinaKa** [comment]
I thought that predict_orthologs was to be used separately too that's 
why I corrected it. But no, I didn't add anything new.

**Comment by jhcepas** [comment]
we could have it, but it looks that many methods will be redundant, so better one single master script and all new functionality in the orthology.py module

**Pull request closed without merging**